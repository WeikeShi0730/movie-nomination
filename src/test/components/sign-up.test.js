import React from "react";
import { shallow } from "enzyme";
import SignUp from "../../components/sign-up/sign-up.component";

it("should render SignUp component", () => {
  expect(shallow(<SignUp />).getElement()).toMatchSnapshot();
});
