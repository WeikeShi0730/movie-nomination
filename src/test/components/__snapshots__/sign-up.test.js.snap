// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignUp component 1`] = `
<div
  className="sign-up"
>
  <h2
    className="title"
  >
    I do not have an account
  </h2>
  <span>
    Sign up with your email and password
  </span>
  <form
    className="sign-up-form"
    onSubmit={[Function]}
  >
    <FormInput
      label="Display Name"
      name="displayName"
      onChange={[Function]}
      required={true}
      type="text"
      value=""
    />
    <FormInput
      label="Email"
      name="email"
      onChange={[Function]}
      required={true}
      type="email"
      value=""
    />
    <FormInput
      label="Password"
      name="password"
      onChange={[Function]}
      required={true}
      type="password"
      value=""
    />
    <FormInput
      label="Confirm Password"
      name="confirmPassword"
      onChange={[Function]}
      required={true}
      type="password"
      value=""
    />
    <CustomButton
      type="submit"
    >
      Sign Up
    </CustomButton>
  </form>
</div>
`;
